runtime/src/accounts_db.rs

```diff
@@ -4272,6 +4272,52 @@ impl AccountsDb {
         }
     }
 
+    /// Perform the scan for dirty pubkeys in a slot
```

Comment:
nit: `dirty` ->`pubkeys that were written to` to be more explicit

Reply:
Done!